packages/tools/pluggable-widgets-tools/src/typings-generator/generateClientTypes.ts

```diff
@@ -13,8 +18,8 @@ ${generateClientTypeBody(properties, true, results)}
             : `export interface ${widgetName}ContainerProps {
     name: string;
     class: string;
-    style?: CSSProperties;
-    tabIndex: number;
+    ${isLabeled ? `id: string;` : `style?: CSSProperties;`}
```

Comment:
Please leave `style?: CSSProperties;` in place unconditionally, since there is no reason to remove it. Plus we might start exporting `style` for labeled widgets in the future.

Reply:
Plus this change might break existing widgets.

Reply:
Also would be nice to put `id` after `tabIndex`, to keep the always-present part of properties on top.

Reply:
How does break existing widgets when its optional? It is already? so if you assume its there... its breaks..
Restored it any way for future use.